src/main/antlr/org/osiam/storage/parser/LogicalOperatorRules.g4

```diff
@@ -24,6 +24,10 @@
 /** Define a grammer called LogicalOperatorRules which will be the basis for ARNTL to generate the java classes for lexer and parser */
 grammar LogicalOperatorRules;
 
+@header {
+package org.osiam.storage.parser;
+}
```

Comment:
The Gradle antlr plugin does a little less magic than the maven one. Which I consider a good thing.